src/components/Comment/Comment.tsx

```diff
@@ -455,9 +460,11 @@ export const Comment = ({
 										</div>
 									</Row>
 									<Row>
-										{showStaffbadge ? (
+										{showStaffBadge || showContributorBadge ? (
```

Comment:
This feels a bit like these two bits of logic are fighting each other. Maybe we can break this apart and simply have

```
{showStaffBadge && <GuardianStaff role='Staff' />}
{showContributorBadge && <GuardianStaff role='Contributor' />}
```

?

If we explicitly want to prevent both badges showing at the same time you could


```
{showStaffBadge && <GuardianStaff role='Staff' />}
{showContributorBadge && !showStaffBadge && <GuardianStaff role='Contributor' />}
```

not sure if this extra check is needed or not. Ideally the server would decide this type of thing and the rendering layer just does what it is asked to do